### DISCUSSION TITLE
Is there a way to change the talk settings permanently?

### DISCUSSION THREAD

**Discussion by ladislavkrivy** [nextcloud/all-in-one#959] [Questions] [answered]
Changes to the talk settings (eg. adding TURN server) get reverted when the container starts. Ideally I'd like to modify the talk config but don't know how/where.

Reason: I'm exposing the AIO server from behind NAT using a VPS. That adds significant latency. I don't want to use thesignalling backend, just an external STUN/TURN server. Not forwarding port 3478 and running coturn server with the original secret on the VPS works fine but I have to disable signalling server after every downtime.

**Comment by ladislavkrivy** [comment]
Ugly ugly workaround for the time being is using cron to run occ command to un-set signalling server.
`docker exec nextcloud-aio-nextcloud php occ talk:signaling:delete https://mydomain.tld/standalone-signaling/`

…related question - is there any way to run hook script when nextcloud container starts?

**Comment by szaimen** [comment, marked as answer]
See https://github.com/nextcloud/all-in-one/discussions/1088 for the feature request
